src/components/Settings/Settings.tsx

```diff
@@ -110,6 +120,14 @@ const SettingsModal: React.FC<ModalProps> = ({
             </div>
           )}
 
+          {activeTab === "textGeneration" && (
```

Comment:
Nit: See this as another repeat of the above, ideally we use standard enums for magic-ish strings.